packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js

```diff
@@ -49,13 +49,9 @@ export default class OvhPaymentMethodIntegrationComponentAdyenCtrl {
 
   initializeComponent() {
     const adyenConfiguration = {
-      onConfigSuccess: () => {
-        this.$timeout(() => {
-          this.isComponentLoaded = true;
-        });
-      },
       onChange: (state) => {
         this.$timeout(() => {
+          this.isComponentLoaded = true;
```

Comment:
No more `onConfigSuccess` callback. `onChange' is triggered when config succes is OK